baseclasses/pyAero_problem.py

```diff
@@ -600,18 +730,24 @@ def setDesignVars(self, x):
 
         for dvName in self.DVs:
             if dvName in x:
-                key = self.DVs[dvName].key
-                family = self.DVs[dvName].family
-                value = x[dvName] + self.DVs[dvName].offset
-                if family is None:
-                    setattr(self, key, value)
+                dv = self.DVs[dvName]
+                key = dv.key
+                family = dv.family
+                value = x[dvName] + dv.offset
+                if key in self.possibleBCDVs:
+                    self.bc_data[family][key] = value
+
+                elif key in self.possibleActuatorDVs:
+                    self.actuatorData[family][key] = value
                 else:
-                    self.bcVarData[key, family] = value
+                    setattr(self, key, value)
```

Comment:
just a clarification on the "original" code: what vars would we be adding here? I assume there are previous check on the `key` to be sure we are not adding a random name/var